.github/workflows/python_tests.yml

```diff
@@ -11,8 +11,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        platform: [ubuntu-latest, windows-latest]  # macos-latest
-        python-version: ["3.9", "3.10", "3.11"]
+        platform: [ubuntu-latest, windows-latest]  # macos-latest disabled because of lightgbm fail
```

Comment:
Oh, I did not realize that YASA fails on mac... too bad, I wonder if there is a way around this in the future. Do you have any ideas on that? Just asking here but of course nothing to address for this PR 👍

Reply:
Problem solved — tests are now passing on macos as well 🚀